docs/conceptual/core/consensus.md

```diff
@@ -2,14 +2,104 @@
 title: Consensus
 ---
 
-* What is consensus for on a blockchain?
+## State Machines and Conflicts
+A blockchain runtime is a [state machine](https://en.wikipedia.org/wiki/Finite-state_machine). It has some internal state, and state transition function that allows it to transition from its current state to a future state. In most runtimes there are states that have valid transitions to multiple future states, but a single transition must be selected. Consider the following examples.
 
-* Block Production
+### Voting Conflict
+A blockchain runtime tallys simple two-option votes for a  DAO. When a new election is opened, Alice, a member of the DAO, can vote Aye. Alice could also vote Nay. But Alice cannot vote both Aye and Nay in the same election.
 
-* Finality
+### Land Registry Conflict
```

Comment:
Can this conflict be explained without "Land Registry"? Something more generic?

Reply:
Land registry is an example. IT sits alongside the voting example and the double spend example.